src/index.jsx

```diff
@@ -1,11 +1,32 @@
 import 'core-js/fn/object/assign';
 import React from 'react';
 import ReactDOM from 'react-dom';
+// Import routing components
+import { Router, Route, browserHistory } from 'react-router';
 
-import App from './components/app';
+var data = [
+	{
+	info1: 'Tired',
+	info2: 'More Awake'
+	},
+	{
+	info1: 'Very Tired',
+	info2: 'Even More Awake'
+	},
+	{
+	info1: 'Sleeping',
+	info2: 'Buzzin'
+	}]
```

Comment:
Hmm, linter should have thrown this out for a few reasons.  There's no trailing semi-colon and the indentation is inconsistent.  I need to check the linter settings.